cypress/e2e/circulation-log/search.cy.js

```diff
@@ -186,4 +187,22 @@ describe('circulation-log', () => {
     SearchPane.searchByChangedDueDate();
     SearchPane.verifyResultCells();
   });
+
+  it('C16997 Filter circulation log by Claimed returned (firebird)', { tags: [TestTypes.criticalPath, devTeams.firebird] }, () => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
```

Comment:
And use cy.loginAsAdmin() instead of simple login

Reply:
done